ecosystem.toml

```diff
@@ -10,6 +10,10 @@ description = "to be written"
 name = "REPL"
 description = "REPLs are a common way to interact with compilers by executing code statement by statement."
 
+[topics.runtime]
+name = "Runtimes"
```

Comment:
I'd elaborate and specifically call it "Code Generation", although `codegen` could be used for the slug (i.e. `topics.codegen`).

Reply:
I would argue that since you have to lower the semantic to the cranelift/llvm abstract machine that runtime is a good enough general name for both codegen backends and interpreters like HVM

Reply:
Though maybe the word I just used, "backend," would be more accessible.

Reply:
Backend sounds great.

Reply:
eh... just split the 2? HVM can go into runtimes, the rest can go into codegen. Also, as a note - HVM is actually probably more suited to the showcase than the projects list.

Reply:
Last I looked the author didn't really have a lang using HVM, but it looks like they do now
https://github.com/Kindelia/Kind2
idk what the guidelines are supposed to be for showcases. Do we want langs or anything "lang dev" related?

Reply:
Hmm. Let's merge this for now, we can always adjust later.